toolchain/driver/clang_runner.cpp

```diff
@@ -522,7 +674,7 @@ auto ClangRunner::BuildBuiltinsLib(llvm::StringRef target,
       lib_dir.OpenWriteOnly(builtins_a_path, Filesystem::CreateAlways));
   {
     llvm::raw_fd_ostream builtins_a_os = builtins_a_file.WriteStream();
-
+    member_group.wait();
```

Comment:
Does this synchronize `objs` to the current thread or do we need to lock to ensure we see the right values?